python/pipeline/experiment.py

```diff
@@ -590,7 +609,7 @@ class AutoProcessing(dj.Manual):
     -> Scan
     ---
     priority=0          :tinyint       # highest priority is processed first
-    autosegment=false   :boolean       # segment somas in the first channel with default method
+    autosegment=0   :boolean       # segment somas in the first channel with default method
```

Comment:
any reason to make it `0` instead of `false`?

Reply:
i encountered the following error with a default value of `false`.  i am not sure why this was the case.  any suggestions?

```
---------------------------------------------------------------------------
OperationalError                          Traceback (most recent call last)
<ipython-input-2-48638f745193> in <module>
----> 1 from pipeline import pupil

/data/pipeline/python/pipeline/pupil.py in <module>
     37 from .utils.eye_tracking import PupilTracker, ManualTracker
     38 from . import config
---> 39 from . import experiment, notify, shared
     40 from .exceptions import PipelineException
     41 

/data/pipeline/python/pipeline/experiment.py in <module>
    605 
    606 @schema
--> 607 class AutoProcessing(dj.Manual):
    608     definition = """ # scans that should enter automatic processing
    609     -> Scan

/usr/local/lib/python3.6/dist-packages/datajoint/schemas.py in __call__(self, cls, context)
    215         if issubclass(cls, Part):
    216             raise DataJointError('The schema decorator should not be applied to Part relations')
--> 217         self.process_table_class(cls, context=dict(context, self=cls, **{cls.__name__: cls}))
    218 
    219         # Process part relations

/usr/local/lib/python3.6/dist-packages/datajoint/schemas.py in process_table_class(self, table_class, context, assert_declared)
    188                 raise DataJointError('Table `%s` not declared' % instance.table_name)
    189             else:
--> 190                 instance.declare(context)
    191         is_declared = is_declared or instance.is_declared
    192 

/usr/local/lib/python3.6/dist-packages/datajoint/table.py in declare(self, context)
     65             for store in external_stores:
     66                 self.connection.schemas[self.database].external[store]
---> 67             self.connection.query(sql)
     68         except AccessError:
     69             # skip if no create privilege

/usr/local/lib/python3.6/dist-packages/datajoint/connection.py in query(self, query, args, as_dict, suppress_warnings, reconnect)
    212         cursor = self._conn.cursor(cursor=cursor_class)
    213         try:
--> 214             self._execute_query(cursor, query, args, cursor_class, suppress_warnings)
    215         except errors.LostConnectionError:
    216             if not reconnect:

/usr/local/lib/python3.6/dist-packages/datajoint/connection.py in _execute_query(cursor, query, args, cursor_class, suppress_warnings)
    194                 cursor.execute(query, args)
    195         except client.err.Error as err:
--> 196             raise translate_query_error(err, query) from None
    197 
    198     def query(self, query, args=(), *, as_dict=False, suppress_warnings=True, reconnect=None):

/usr/local/lib/python3.6/dist-packages/datajoint/connection.py in _execute_query(cursor, query, args, cursor_class, suppress_warnings)
    192                     # suppress all warnings arising from underlying SQL library
    193                     warnings.simplefilter("ignore")
--> 194                 cursor.execute(query, args)
    195         except client.err.Error as err:
    196             raise translate_query_error(err, query) from None

/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py in execute(self, query, args)
    161         query = self.mogrify(query, args)
    162 
--> 163         result = self._query(query)
    164         self._executed = query
    165         return result

/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py in _query(self, q)
    319         self._last_executed = q
    320         self._clear_result()
--> 321         conn.query(q)
    322         self._do_get_result()
    323         return self.rowcount

/usr/local/lib/python3.6/dist-packages/pymysql/connections.py in query(self, sql, unbuffered)
    503                 sql = sql.encode(self.encoding, 'surrogateescape')
    504         self._execute_command(COMMAND.COM_QUERY, sql)
--> 505         self._affected_rows = self._read_query_result(unbuffered=unbuffered)
    506         return self._affected_rows
    507 

/usr/local/lib/python3.6/dist-packages/pymysql/connections.py in _read_query_result(self, unbuffered)
    722         else:
    723             result = MySQLResult(self)
--> 724             result.read()
    725         self._result = result
    726         if result.server_status is not None:

/usr/local/lib/python3.6/dist-packages/pymysql/connections.py in read(self)
   1067     def read(self):
   1068         try:
-> 1069             first_packet = self.connection._read_packet()
   1070 
   1071             if first_packet.is_ok_packet():

/usr/local/lib/python3.6/dist-packages/pymysql/connections.py in _read_packet(self, packet_type)
    674             if self._result is not None and self._result.unbuffered_active is True:
    675                 self._result.unbuffered_active = False
--> 676             packet.raise_for_error()
    677         return packet
    678 

/usr/local/lib/python3.6/dist-packages/pymysql/protocol.py in raise_for_error(self)
    221         errno = self.read_uint16()
    222         if DEBUG: print("errno =", errno)
--> 223         err.raise_mysql_exception(self._data)
    224 
    225     def dump(self):

/usr/local/lib/python3.6/dist-packages/pymysql/err.py in raise_mysql_exception(data)
    105     if errorclass is None:
    106         errorclass = InternalError if errno < 1000 else OperationalError
--> 107     raise errorclass(errno, errval)

OperationalError: (1067, "Invalid default value for 'autosegment'")
```

Reply:
Interesting, ok then let's set to 0.